### ISSUE TITLE
build: run full tests in a fast builder

### ISSUE THREAD

**Issue by josharian** [golang/go#10456]
all.bash runs with test -short. However, sometimes non-short tests break. See e.g. #10455 or #8617. These get found when someone eventually runs the full tests during development.

It'd be useful to have one fast builder run the full set of tests for each commit.

/cc @bradfitz

**Comment by bradfitz** [comment]
Let's do this once #10029 (sharding over VMs) is done.

**Comment by josharian** [comment]
Duplicate of #12508, which has more discussion.
